spec/api/v1/admin/visits_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+describe V1::Admin::Visits do
+  include AuthHelper
+
+  let!(:current_user) { create :user }
+  let!(:visits) { create_list :visit, 2 }
+
+  describe 'POST /api/v1/admin/visits' do
```

Comment:
I'd avoid writing routes in description:
- If a route's changed in the future, you may forget to fix descriptions in tests;
- you give your controller a global context (a knowledge about its accessing from outside - what breaks good practices)